dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -295,6 +498,8 @@
 
   {{ run_hooks(post_hooks, inside_transaction=True) }}
```

Comment:
I found a bug recently that imply running persisting docs after run_hooks, let's move persist_docs after run hooks, it's really relevant for iceberg. See this https://github.com/dbt-athena/dbt-athena/pull/187